benchmarks/syncing/syncing.rs

```diff
@@ -36,7 +36,8 @@ fn providing_sync_blocks(c: &mut Criterion) {
 
     let blocks = TestBlocks::load(NUM_BLOCKS);
     for block in &blocks.0 {
-        provider.expect_sync().consensus.receive_block(&block).unwrap();
+        rt.block_on(provider.expect_sync().consensus.receive_block(&block))
```

Comment:
would it make sense to wrap the loop in a `task::spawn_blocked`, so that it is processed in one of the threads dedicated to blocking in its entirety?

Reply:
does it matter here? its a prelude for a benchmark run with a controlled runtime.

Reply:
ah yes, didn't notice it was benchmark code :ok_hand: